releasenotes/notes/remove-quantum-instance-and-runtime-2c00393c0de1b9a9.yaml

```diff
@@ -0,0 +1,7 @@
+---
+upgrade:
+  - |
+    Support for ``QuantumInstance``-based algorithms has been removed.
```

Comment:
Perhaps we might like to say that only primitive based algorithms are supported now and link to the 0.5 migration guide. Of course beyond that migrating to qiskit_algorithms too but I guess we'll cover that elsewhere